test/unit/storage_test.rb

```diff
@@ -113,6 +113,35 @@ def test_sentinels_simple_url
                                       sentinels: [{ host: '127.0.0.1', port: 26_379 }])
   end
 
+  def test_sentinels_array_hashes_default_port
+    default_sentinel_port = Storage::Helpers.singleton_class.const_get(:DEFAULT_SENTINEL_PORT)
+    config_obj = {
+      url: 'redis://127.0.0.1:6379/0',
+      sentinels: [{ host: '127.0.0.1' }, { host: '192.168.1.1' },
+                  { host: '192.168.1.2', port: nil }]
```

Comment:
my comment got outdated with your latest push - I was saying that yes, specifying the port is done elsewhere, but there is still no combination in any previous test of specifying the port and not specifying it and testing correct values, so either add it here or add it in some of the previous tests.

Reply:
:+1: I answered you in the outdated comment